CHANGELOG.md

```diff
@@ -11,6 +12,48 @@ Version numbers 0.2.x to 0.7.x were intentionally not used to avoid conflicts wi
 
 **Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
 
+# v0.9.0
+
+**This is the third alpha version of the Quality-On-Demand (QoD) API.**
+
+- [API definition with inline documentation](https://github.com/camaraproject/QualityOnDemand/releases/tag/release-0.9.0/code/API_definitions)
+
```

Comment:
Is that the correct link? It doesn't work for the -rc either (same format).

Reply:
@akoshunyadi this link is correct but will work only after the release (and the release branch) is created.

But the link for within the v0.9.0-rc was wrong, as we haven't created a release branch for the release candidate. Corrected the link so that it uses the now the tag.

Reply:
Actually the link was wrong ... :-(